pkg/volume/persistent_claim/persistent_claim.go

```diff
@@ -58,16 +58,26 @@ func (plugin *persistentClaimPlugin) NewBuilder(spec *volume.Spec, pod *api.Pod,
 		return nil, err
 	}
 
-	if claim.Status.VolumeRef == nil {
+	if claim.Spec.VolumeName == "" {
 		return nil, fmt.Errorf("The claim %+v is not yet bound to a volume", claim.Name)
 	}
 
-	pv, err := plugin.host.GetKubeClient().PersistentVolumes().Get(claim.Status.VolumeRef.Name)
+	pv, err := plugin.host.GetKubeClient().PersistentVolumes().Get(claim.Spec.VolumeName)
 	if err != nil {
 		glog.Errorf("Error finding persistent volume for claim: %+v\n", claim.Name)
 		return nil, err
 	}
 
+	if pv.Spec.ClaimRef == nil {
+		glog.Errorf("Expected volume.Spec.ClaimRef but have nil: %+v", pv)
```

Comment:
This will be a message users see - can it be more helpful?  "The volume %s is not yet bound to the claim" or something